pom.xml

```diff
@@ -107,10 +131,15 @@
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
This plugin is already in plugin management, so shouldn't need to list it here. At least definitely don't need the version listed here.

Reply:
Shall update to remove it